docs/title-and-description.md

```diff
@@ -76,4 +76,55 @@ metadata:
 ```
 
 The above examples will render as rows like the below image:
-![More Markdown Examples](assets/workflow-title-and-description-markdown-complex.png)
+![More Markdown Examples](assets/title-and-description-markdown-complex-workflow.png)
+
+### Cluster Workflow Template
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: my-cluster-workflow-template
+  annotations:
+    workflows.argoproj.io/title: '**Test Title**'
+    workflows.argoproj.io/description: |
+      `This is a simple hello world example.`
+      You can also run it in Python: https://couler-proj.github.io/couler/examples/#hello-world
+```
+
+The above examples will render as rows like the below image:
```

Comment:
```suggestion
The above manifest will render as a row like the below image:
```

singular "example" and singular "row". the existing doc uses "manifest" when there's a `kind` specified, so also use the term "manifest" here as well